src/store/broadcaster/state.js

```diff
@@ -2,15 +2,20 @@ export default function () {
   return {
     data: [
```

Comment:
name this as offices or something that represents its content not generic as data